src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -531,21 +529,25 @@ const AccessibilityRequestDetailPage = () => {
           )}
         </div>
       </div>
-      <div className="grid-container margin-top-2 padding-top-6 padding-top">
+      {isAccessibilityTeam && (
+        <SecondaryNav>
+          <NavLink to={`/508/requests/${accessibilityRequestId}`}>
+            Documents
+          </NavLink>
+          <NavLink to={`/508/requests/${accessibilityRequestId}/notes`}>
+            Notes
+          </NavLink>
+        </SecondaryNav>
+      )}
+      <div className="grid-container padding-top-6 padding-top">
         <div className="grid-row grid-gap-lg">
           <div className="grid-col-8">
-            {isAccessibilityTeam ? (
-              <Tabs>
-                <TabPanel id="Documents" tabName="Documents">
-                  <div className="margin-top-2">{bodyWithDocumentsTable}</div>
-                </TabPanel>
-                <TabPanel id="Notes" tabName="Notes">
-                  {notesTab}
-                </TabPanel>
-              </Tabs>
-            ) : (
-              documentsTab
-            )}
+            {/* eslint-disable-next-line no-nested-ternary */}
+            {isAccessibilityTeam
```

Comment:
nit: I find nested ternaries confusing. I think it can be separated into two, maybe? This is the least of our worries tho.